examples/step-10/step-10.cc

```diff
@@ -131,22 +131,24 @@ namespace Step10
             // face is drawn by 30 small lines instead of only one. This is
             // sufficient to give us the impression of seeing a curved line,
             // rather than a set of straight lines.
-            GridOut               grid_out;
+            GridOut grid_out;
             GridOutFlags::Gnuplot gnuplot_flags(false, 60);
             grid_out.set_flags(gnuplot_flags);
 
             // Finally, generate a filename and a file for output:
-            std::string filename = filename_base + "_mapping_q_" +
-                                   Utilities::to_string(degree) + ".dat";
-            std::ofstream gnuplot_file(filename);
+            std::string filename = filename_base
+                                   + "_mapping_q_"
+                                   + Utilities::to_string(degree)
+                                   + ".dat";
```

Comment:
That's another instance, where not smuching everything into as few lines as possible looks better:
```
std::string filename = filename_base                  //
                       + "_mapping_q_"                //
                       + Utilities::to_string(degree) //
                       + ".dat";                      //
```